2016/playbook/session_3/experiments/examples_for_inspiration.md

```diff
@@ -1,14 +1,32 @@
 # Providing better examples to further increase hacker inspiration
 
-## Implementation 
+## July 20
+
+### Implementation 
 
 Mentors will find examples for projects students have created from the workshop to increase hacker inspiration.
 
-## Observations 
+### Observations 
 
 - People had interesting projects today, and it didn't seem like anyone was blocking on inspiration and/or creativity.
 - Saw a lot of people looking at projects that were linked below regardless.
 
-## Conclusion
+### Conclusion
 
 Unclear whether or not the examples helped hackers with creativity and inspiration, trying experiment again tomorrow to see what happens. 
+
+## July 21
+
+### Implementation
+
+Mentors will find examples for projects students have created from the workshop, or find online projects (ie online games) to increase hacker inspiration.
+
+### Observations
+
+- Most of the examples already provided in the workshop weren't very original, that is, they closely resembled the workshop.
+- Many of the hackers didn't have projects that actually deviated from the original Dodge workshop we had, most of it was just different player sprites and enemies. 
+- PR with the online game example wasn't merged in, so I'm pretty sure most of the hackers didn't see the example.  Also, mentors didn't really see too many students looking at the examples. 
```

Comment:
Because these docs don't have your name directly on them, could you put this in the third person?